packages/axios-extension/src/abap/ui5-abap-repository-service.ts

```diff
@@ -126,7 +127,10 @@ export class Ui5AbapRepositoryService extends ODataService {
             const response = await this.get<AppInfo>(`/Repositories('${encodeURIComponent(app)}')`);
             return response.odata();
         } catch (error) {
-            this.log.debug(`Retrieving application ${app}, error found ${error}`);
+            this.log.debug(`Retrieving application ${app}, ${error}`);
+            if (isAxiosError(error) && error.response?.status !== 404) {
```

Comment:
We only want to return `undefined` if it is a 404 otherwise, we don't know what to do with the error, therefore, it should be 
```
if (isAxiosError(error) && error.response?.status === 404) {
   return undefined;
} else {
   throw error;
}
```